wordpress.org/public_html/wp-content/plugins/wporg-bbp-also-viewing/wporg-bbp-also-viewing.js

```diff
@@ -11,6 +11,8 @@
 		banner = false,
```

Comment:
Not a blocker, just a suggestion— Could you create a block for this? maybe with the Interactivity API? (the declarative structure of the interactivity api might not work nicely with the setInterval calls). Rather than checking for `$localNavBar` to split the behavior, the block theme would use the block version, and the jQuery version could be deprecated.

That also might be more trouble than it's worth, and can be revisited if/whenever we fully blockify this :)

Reply:
Yeah great idea. Ticket created https://meta.trac.wordpress.org/ticket/7503